python/ray/tune/experiment/trial.py

```diff
@@ -186,6 +191,58 @@ def _create_unique_logdir_name(root: str, relative_logdir: str) -> str:
     return relative_logdir
 
 
+def _noop_logger_creator(
+    config: Dict[str, Any], logdir: str, should_chdir: bool = True
+):
+    # Upon remote process setup, record the actor's original working dir before
+    # changing to the Tune logdir
+    os.environ.setdefault("TUNE_ORIG_WORKING_DIR", os.getcwd())
+
+    os.makedirs(logdir, exist_ok=True)
+    if should_chdir:
+        # Set the working dir to the trial directory in the remote process,
+        # for user file writes
+        if not ray._private.worker._mode() == ray._private.worker.LOCAL_MODE:
+            os.chdir(logdir)
+    return NoopLogger(config, logdir)
+
+
+def _get_trainable_kwargs(
+    trial: "Trial",
+    additional_kwargs: Optional[Dict[str, Any]] = None,
+    should_chdir: bool = False,
+) -> Dict[str, Any]:
+    trial.init_logdir()
+
+    logger_creator = partial(
+        _noop_logger_creator,
+        logdir=trial.logdir,
+        should_chdir=should_chdir,
+    )
+
+    trial_config = copy.deepcopy(trial.config)
+    trial_config[TRIAL_INFO] = _TrialInfo(trial)
+    stdout_file, stderr_file = trial.log_to_file
+    trial_config[STDOUT_FILE] = stdout_file
+    trial_config[STDERR_FILE] = stderr_file
+
+    kwargs = {
+        "config": trial_config,
+        "logger_creator": logger_creator,
+    }
+
+    if trial.uses_cloud_checkpointing:
+        # We keep these kwargs separate for backwards compatibility
+        # with trainables that don't provide these keyword arguments
+        kwargs["remote_checkpoint_dir"] = trial.remote_checkpoint_dir
+        kwargs["sync_config"] = trial.sync_config
+
+        if additional_kwargs:
+            kwargs.update(additional_kwargs)
```

Comment:
Should this be included regardless of cloud checkpointing or not?

Reply:
I'm keeping the original logic intact, which only applied this when cloud checkpointing was enabled. Because we'll remove this anyway I think we can keep it as is for now.